3.10/transactions.md

```diff
@@ -1,30 +1,64 @@
 ---
 layout: default
-description: ArangoDB provides support for user-definable  transactions
+description: ArangoDB provides support for user-definable transactions
 ---
-Transactions
-============
+# Transactions
 
-ArangoDB provides support for user-definable transactions.
+## Transaction Types
+
+ArangoDB offers different types of transactions:
+
+- AQL queries (with exceptions)
+- Stream Transactions
+- JavaScript Transactions
+
+<!-- TODO
+### AQL Queries
+
+read own writes (UPSERT?), intermediate commits
+-->
+
+### Stream Transactions
```

Comment:
According to the design doc, they would need to be called El Cheapo Transactions:
https://github.com/arangodb/documents/blob/9eddec21611539df26e301ef7de88a2176ea344a/DesignDocuments/04_IMPLEMENTED/ElCheapoTransactions.md

However, Simon went with Stream Transactions in the docs PRs, e.g. here:
https://github.com/arangodb/arangodb/pull/8833

Therefore, I consistently used this term in the docs. Others called it Streaming Transactions, however, in PRs as well as in our release announcement:
https://www.arangodb.com/2019/08/multi-model-database-3-5-released-distributed-joins-transactions-graphdb-search/

To increase consistency, we can rename the startup options (the previous names will continue to work). Or rename it everywhere in the docs and set up redirects because it is used in filenames / URLs.

BTW. There is a similar disagreement about stream cursors / streaming cursors / streaming queries (the cursor option is called `stream`). It would be helpful to capture PM's authoritative naming decisions somewhere...

Reply:
We will stick to Stream Transactions in docs, at least for the time being.